server/backend/database/memory/database.go

```diff
@@ -786,6 +828,35 @@ func (d *DB) FindClosestSnapshotInfo(
 	return snapshotInfo, nil
 }
 
+// FindMinSyncedSeqInfo finds the minimum synced sequence info.
+func (d DB) FindMinSyncedSeqInfo(
```

Comment:
```suggestion
func (d *DB) FindMinSyncedSeqInfo(
```

Reply:
There was a fatal typo...
I'll update it.